MainDemo.Wpf/Fields.xaml

```diff
@@ -339,9 +339,14 @@
                         Grid.Column="3"
                         VerticalAlignment="Bottom"
                         UniqueKey="passwordFilled1">
-        <PasswordBox materialDesign:HintAssist.HelperText="Helper text"
-                     materialDesign:HintAssist.Hint="Password"
-                     Style="{StaticResource MaterialDesignFilledPasswordBox}" />
+        <StackPanel>
+          <CheckBox x:Name="MaterialDesignFilledPasswordBoxTextCountComboBox" Content="View Pwd Input Count" />
```

Comment:
I would spell out password
```suggestion
          <CheckBox x:Name="MaterialDesignFilledPasswordBoxTextCountComboBox" Content="View Password Input Count" />
```

Reply:
Did that initially, but then it clips in the demo tool with the current width of the ShowMeTheXAML elements (fixed width). So I took the shortcut and renamed 😀